fe/fe-core/src/main/java/com/starrocks/catalog/system/information/LoadsSystemTable.java

```diff
@@ -52,6 +52,7 @@ public static SystemTable create() {
                         .column("TRACKING_URL", ScalarType.createVarchar(NAME_CHAR_LEN))
                         .column("TRACKING_SQL", ScalarType.createVarchar(NAME_CHAR_LEN))
                         .column("REJECTED_RECORD_PATH", ScalarType.createVarchar(NAME_CHAR_LEN))
+                        .column("WAREHOUSE", ScalarType.createVarchar(NAME_CHAR_LEN))
```

Comment:
‌‌I suggest that we should maintain consistency with the previous logic, and add the display of this column when the cluster is running in shared-data mode.